src/Sylius/Bundle/PaymentBundle/Resources/config/app/state_machine.yml

```
@@ -23,14 +24,17 @@ winzou_state_machine:
             process:
                 from: [new]
                 to: processing
-            complete:
+            authorize:
                 from: [new, processing]
+                to: authorized
+            complete:
+                from: [new, processing, authorized]
                 to: completed
             fail:
                 from: [new, processing]
```

```suggestion
                from: [new, processing, authorized]
```
I'd assume a payment could fail at basically any point.
Similarly with `void`.

---

Ok :sunglasses: done